### DISCUSSION TITLE
when release for beta 10.0 ?

### DISCUSSION THREAD

**Discussion by element221** [Hekili/hekili#1621] [Q&A]
I'm wondering when there will be a release for beta, so we can test both the game and add-on.

**Comment by Hekili** [comment]
**DEVELOPMENT UPDATES**
For any works-in-progress, please refer to <https://github.com/Hekili/hekili/releases> to find anything I have provided in a shareable state.  For a quick summary, see below.

**DRAGONFLIGHT**
Development is well underway, but there is not currently an alpha/beta/release version of the addon Dragonflight beta or PTR.  When it is released, you will be able to find it at the above link.

One key factor is that there really aren't testable priority lists available for any specs as of now (9/21/2022).  This addon generally presents recommendations based on SimulationCraft priorities which are not complete at this time.

Before the addon is really ready for any meaningful use, I have to be able to accomplish all of the following for each spec.

1. Resource Modeling
2. Talents
3. PvP Talents
4. Auras
5. New/Changed Abilities
6. Duplicate Abilities
7. Options
8. Priority Lists

For most specializations, 2 - 4 are mostly covered.  #1 may *mostly* carry over from Shadowlands.  #5 may also carry over, except new abilities have to be scripted from scratch.  #6 is largely an issue of handling when people are still using Covenant abilities that become talents in Dragonflight.

All of the previous stuff is needed to assess #s 7 and 8.  tl;dr:  A lot is done, but there's a long way to go.

**WRATH OF THE LICH KING**
I have continued to do research on Wrath of the Lich King support, and I know there's a lot of enthusiasm around having the addon for Wrath.  I have been doing research using SimC from 3.3.5 and may be able to *mostly* automate the class modules for Wrath.  If I can do so, and release a Wrath version (or one version of the addon for Retail + Wrath) that doesn't add a tremendous amount of maintenance to the overall addon, I will continue with that effort.

If/when there is a release that supports Wrath, I'll post about it.  In the meantime, questions will only take more time away from the research and development of the addon.
